src/webgl/interaction.js

```diff
@@ -19,19 +19,30 @@ var p5 = require('../core/core');
  * <code>
  * function setup() {
  *   createCanvas(100, 100, WEBGL);
+ *   normalMaterial();
  * }
- *
  * function draw() {
  *   background(200);
- *   // Orbit control allows the camera to orbit around a target.
  *   orbitControl();
- *   box(30, 50);
+ *   translate(-20, 15, -20);
```

Comment:
the original example here was a little simpler. did you think it didn't illustrate the concept clearly enough? could this more complex example be a second example?

Reply:
I reverted to the original.  I was a bit concerned it would be difficult to see the panning behavior with only one object in the scene, but on second look, it doesn't seem to be a problem.